resources/user-guides/mining_with_xmrig_and_docker.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: static_page
+title: "Mining with XMRig and Docker"
+title-pre-kick: "Mining XMR"
+title-kick: "With XMRig and DOcker"
+title-post-kick: ""
+kick-class: "purple-kicks"
+icon: "icon_userguides"
+attribution: "<!-- Icon is based on work by Freepik (http://www.freepik.com) and is licensed under Creative Commons BY 3.0 -->"
+---
+
+##Introduction
+
+This guide is two fold, ease of use for mining on Linux distributions and some extra security around mining as most of these miners have not had security auditing.
+
+At the end of this guide you will be able to sleep a little easier knowing that if the miner gets exploited it will not migrate to your OS.
+
+###Why Docker
+
+Docker is being used as it is the most well known and has the biggest chance to be already installed.
```

Comment:
Maybe you could link to the Docker documentation for people that don't already have it installed.